src/test/scala/mesosphere/mesos/ResourceMatcherTest.scala

```diff
@@ -24,7 +26,13 @@ import org.apache.mesos.Protos.Attribute
 import org.apache.mesos.{ Protos => Mesos }
 import org.scalatest.Inside
 
+import scala.concurrent.duration.Duration
+
 class ResourceMatcherTest extends UnitTest with Inside {
+
+  implicit val clock = Clock.systemUTC()
+  val config = AllConf.withTestConfig("--draining_seconds", "300")
```

Comment:
why this? isn't the feature disabled by default?

Reply:
It is off by default, removed configuration